exchanges/btse/btse_test.go

```diff
@@ -752,6 +752,30 @@ func TestStripExponent(t *testing.T) {
 	assert.ErrorIs(t, err, errInvalidPairSymbol, "Should error on a symbol with too many underscores")
 }
 
+func TestMarketPair(t *testing.T) {
+	t.Parallel()
+
+	for _, test := range []struct {
+		symbol         string
+		base           string
+		quote          string
```

Comment:
Super-Nit: Think you could drop this if you wanted, and just put currency.USD inside the loop.